src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Set<Employee> employeeSet = new TreeSet<>(employeeComparator());
+        employeeSet.addAll(employees);
+        return employeeSet;
+    }
+
+    private Comparator<Employee> employeeComparator() {
+        return (o1, o2) -> {
+            int result = o1.getAge() - o2.getAge();
```

Comment:
Direct subtraction to compare ages might result in an integer overflow if the ages are very large. It's safer to use Integer.compare method to avoid this potential issue.